src/module_1/module_1_meteo_api.py

```diff
@@ -70,16 +84,25 @@ def get_data_meteo_api(city, from_year, until_year):
     print("You reached the number of maximum attempts. Stopping the execution")
     sys.exit()
 
+
 def clean_data(raw_data):
+    """
+    (Not used)
```

Comment:
why "not used"? lol

Reply:
I started manipulating dicts but later on found out that it wasn't a good idea, so I decided to leave it there in case it were useful. So I will delete the full parts not finally used